userprofile.css

```diff
@@ -0,0 +1,63 @@
+.profile-container{
+    min-height: 70vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.profile{
+    width: 100%;
+    max-width: 50rem;
+    margin: 2rem;
+    padding: 1rem;
+    border: 2px solid var(--primary-color);
+    border-radius: 5px;
+}
+.profile-btn-container{
+    display: flex;
+    justify-content: center;
+    margin: 0 2rem;
+    padding: 0 0 1rem 0;
+    border-bottom: 2px solid var(--primary-color);
+}
+.profile-btn-wrapper{
+    width: 15.6rem;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+.profile-btn-wrapper button{
+    width: 7.5rem;
+    margin: 0.5rem 0;
+}
+.user-profile-container{
+    display: grid;
+    grid-template-columns: 14.3rem 1fr;
+    margin: 1rem 0 0 0;
+}
+.user-info-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-width: 28.1rem;
```

Comment:
Suggestion: can use only margin-top: 1rem as it has only one top property

Reply:
Done